tests/builtins/conftest.py

```diff
@@ -71,12 +65,6 @@ def queue(sqs):
     return q
 
 
-@pytest.fixture
-def statedb(dynamo, statedb_schema):
-    dynamo.create_table(**statedb_schema)
-    return statedb_schema["TableName"]
-
-
```

Comment:
This doesn't seem right...I replaced the statedb fixture, but it should still be required here. See https://github.com/cirrus-geo/cirrus-geo/blob/a9e25850c29c2647759ac8ca922083f3c32fc6f3/tests/builtins/conftest.py#L70.

Maybe a merge conflict?

This also means the dynamo mock above shouldn't be removed.

Reply:
Never mind, the file sorting confused me when I was looking for the top-level conftest.py and I didn't see it. Ignore. :)